.gitignore

```diff
@@ -0,0 +1,4 @@
+# Idea specifics
+.idea
+git-standup.iml
+
```

Comment:
I think that IDE files must be globally gitignore'd, not in each project `.gitignore` file.